source/kraken/worker.cpp

```diff
@@ -1118,7 +1138,7 @@ void Worker::nearest_stop_points(const pbnavitia::NearestStopPointsRequest& requ
     entry_point.streetnetwork_params.max_duration = navitia::seconds(request.max_duration());
     street_network_worker->init(entry_point, {});
     // kraken don't handle reverse isochrone
-    auto result = routing::get_stop_points(entry_point, *data, *street_network_worker, false);
+    auto result = routing::get_stop_points(entry_point, *data, *street_network_worker, 0u);
```

Comment:
It will work the same way it used to, but this change looks too big to be the initial intent of the developer.

Reply:
This is a mistake from the developer since the last (optional) argument of get_stop_points are an unsigned int (0u by default) and a boolean (false by default).